docs/development-roadmap.stories.mdx

```diff
@@ -0,0 +1,36 @@
+<Meta
+  title="Documentation/Development Roadmap"
+/>
+
+# Sage Carbon public roadmap
+
+> **Disclaimer**: We operate in a dynamic commercial environment, and things are subject to change. The information provided is intended to outline the general direction of the library. It's intended for informational purposes only. We may decide to add/remove new items at any time depending on our capability to deliver while meeting our quality standards. The development, releases, and timing of any features or functionality of Carbon remain at the sole discretion of Sage. The roadmap does not represent a commitment, obligation, or promise to deliver at any time.
+
+## Upcoming activity
+Listed below is the activity we have on our backlog. 
+### Move experimental folder
+Move the experimental components into the main components folder and remove the experimental folder
+### Internationalisation 
+Implement the proposal in the [i18n RFC](https://github.com/Sage/carbon/blob/master/rfcs/text/i18n.md)
+### TypeScript Support 
+ At a minimum we should support TypeScript by providing typings files for every component and enabling support in our build pipeline. 
+We will also review if we should migrate the Carbon repo to use TypeScript completely as there are many benefits to using TypeScript.
+### Accessibility audit
+Review current components against accessibility guidelines and plug any gaps we may have to meet those requirements, in particular: 
+
+1. Keyboard accessibility of navigation components such as Menu. 
+1. Keyboard accessibility of Tables - particularly tables with inputs. 
+1. Focus behaviour in Dialog and Sidebar. 
+1. Date picker popover. 
+
+Automation/Testing/Linting to ensure that Accessibility is part of the CI process so it can’t get forgotten about or worse without us knowing. 
+### Design System congruence
+Review our Design System documentation and ensure our components are congruent with those described in [brand.sage.com](https://brand.sage.com).
+### Deprecate shouldComponentUpdate decorator
+Replace with a higher order component
```

Comment:
I don't think we should be supporting this any more, it's a helper for ImmutableJS which we no longer use.